### ISSUE TITLE
400 imagine payload issue? (RESOLVED)

### ISSUE THREAD

**Issue by charlesknapp** [Wildric-Auric/MidJourney-Wrapper#51]
It seems to happen 50% of the time using the /mj_imagine command. It wasn't happening before this evening, so I'm not sure whats going on.

{"code": 50035, "errors": {"data": {"_errors": [{"code": "INTERACTION_APPLICATION_COMMAND_INVALID_PERMISSIONS", "message": "You do not have permission to use this command"}]}}, "message": "Invalid Form Body"}

**Comment by Wildric-Auric** [comment]
Maybe something has changed again, thanks for reporting this, I will examine it and commit next week maybe.

**Comment by charlesknapp** [comment]
I ended up figuring out what was going wrong with this specific instance.

- I had set specific control permissions in the Discord UI for the bots within my server. By disabling the Midjourney access to one of the tokens I use to generate in my list of active subs, they didn't have access to the Midjourney bot and that was throwing that permissions error.

Took a look at the payload and it didn't seem to change, other than maybe a nonce?
